src/cmd_line.rs

```diff
@@ -120,6 +147,14 @@ pub struct CmdLineSettings {
         default_value = "neovide"
     )]
     pub x11_wm_class_instance: String,
+
+    /// Set background to light, dark, or match system setting with "auto"
+    #[arg(
+        long = "theme",
+        env = "NEOVIDE_THEME",
+        value_parser = EnumValueParser::<ThemeChoice>::new())
+    ]
+    pub theme: ThemeChoice,
```

Comment:
Since you didn't specify a default, the argument will be _always_ required. I suppose you could use `default_value = ThemeChoice::Unset` in the attribute.

Reply:
Thanks for the tip with `Option<>`, it did the trick!